homework/01LessonOne.js

```diff
@@ -202,13 +209,24 @@ var stringFourStrictNotEqualsNumberFour;
 //
 // What’s the difference between == and ===?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: == and === are comparison operators. == is the loose equality operator. It compares two values for equality after it converts both values to a common type.
 //
+// === is the strict equality evaluator. It compares to values and does not convert them. It will produce true if it evaluates to values to be of equal value AND equal type.
 //
 // What’s the difference between != and !==?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: != and !== are comparison operators. != can be considered a "loose not equal to" operator. It evaluates two values and produces a result of true if they are not of equal value.
+// For example:
+// var x = '8';
+//
+// x != '8' will return false because 8 and '8' are considered equal value and '8' is converted into 8.
+//
+//Since ! evaluates if something is not true  8 == 8 being true makes 8 != 8 false.
+//
+// x !== '8' will return true because !== evaluates value and type. x has been assigned the number 8 while '8' is a string.
 //
+//!== does not convert type so any comparison with two different types is automatically considered not equal.
+//x !== '8' is saying that "x is not equal to '8' and is a true statement. //
 // ****
 
```

Comment:
Great work on Homework1! I see all the tests passing. Lets take a look at 01PigLatin....

![pig](http://66.media.tumblr.com/tumblr_lmafhoATpo1qd4wkzo1_400.gif)